app/models/profile.rb

```diff
@@ -43,6 +43,7 @@ def activate
 
   def deactivate(reason)
     update!(active: false, deactivation_reason: reason)
+    UserAlerts::AlertUserAboutAccountRejected.call(user) if reason == :threatmetrix_review_rejected
```

Comment:
Why put this here instead of in the rake task where we are rejecting the user? A concern I have with it in here is it obscures a side-effect of reject the user that a user of the rake task may want to know about.

Reply:
Hmm. My thought in placing it here was that this is the actual place where the deactivation happens. Any changes to the way in which we deactivate users (from a UI perspective) will still send the notification.

Reply:
Modified `Profile#deactivate` to take an argument indicating whether to send the user a notification, and modified all the calls except the rake task's to pass `false`.